pages/selfCheckQuestions.tsx

```diff
@@ -0,0 +1,322 @@
+import React from "react";
+import {
+    Spacer,
+    Text,
+    Modal,
+    ModalBody,
+    ModalContent,
+    ModalOverlay,
+    ModalHeader,
+    ModalFooter,
+    ModalCloseButton,
+    useDisclosure,
+    Flex,
+    SimpleGrid,
+    Button,
+    ButtonGroup,
+} from "@chakra-ui/react";
+import { Header, SelfCheckQuestionCard, Footer } from "@components";
+const questionList = [
```

Comment:
We should try to remove this maybe? Can't see this data on the frontend either

Reply:
> The UI looks great! I left a few comments, will go through more of the PR soon
> 
> Small bug(?) but the console log on save does not show me the correct user input for any title/question values I added. I can see the number of questions and type updating, but the actual content remains the same always

This should be fixed now